src/AWS/CRT/Auth/SigningConfigAWS.php

```diff
@@ -23,6 +23,7 @@ public static function defaults() {
             'signed_body_value' => null,
             'signed_body_header_type' => SignedBodyHeaderType::NONE,
             'expiration_in_seconds' => 0,
+            'date' => time()
```

Comment:
Since this is a unix date, it could be named unixDate or something depending on how it is used to make it clear it has the exact timestamp?

Reply:
so while it is a timestamp, it will be quantized to the year/month/day, so it _is_ a date. In the [SigV4 docs](https://docs.aws.amazon.com/general/latest/gr/sigv4-calculate-signature.html): 
> Note that the date used in the hashing process is in the format YYYYMMDD (for example, 20150830), and does not include the time.